src/index.html

```diff
@@ -34,36 +33,36 @@
       <section class="products">
         <ul class="product-grid">
           <li>
-            <a href="product-listing/index.html?category=tents"
+            <a href="/product-listing/index.html?category=tents"
```

Comment:
It's best practice to prefix you're links with `/`. This makes the link take you to the correct place no matter where you are in the page. Otherwise the link is relative and will act wacky.